docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+
+Bingo! The result should be several files with `outfit_silviocarusso_actor_vX.entitytemplate].pc_entityblueprint` in the name, in both `chunk1` and `chunk8`.
+
+## After Finding the Correct Files
+As we're specifically looking for Caruso's TEMP file, we can ignore the TBLU results. Since the TEMP files in `chunk1patch2` and `chunk8` seem to be duplicates, we can use either of them. But for the sake of this tutorial, let’s use the TEMP files located in `chunk1patch2`.
+Expanding the TEMP tree under `chunk1patch2` shows us that Silvio Caruso has two outfit variants, as indicated by `_v0` and `_v1` in the filenames. Remember that these TEMP files contain references to the components that are used for a certain outfit. Therefore, by selecting the `v1` variant and checking the `Depends on X other hash files/resources` section in the panel on the right, we can see what the outfit consists of. In this case, we can see the entry:
+
+`[assembly:/_pro/characters/assets/workers/scientist_01/geometry/malereg_`**hazmatsuit**`_01.wl2?/shoes.weightedprim](bodypart).pc_weightedprim`
+
+This shows us that the v1 variant is Silvio Caruso in a hazmat suit. Therefore, v0 must be his standard suit simply because it's the only other option. However, to make sure, we can select the v0 TEMP file in the left box, and examine the outfit components again in the right box. v0 makes no mention of a hazmat suit, but does reference other components we know to be part of Caruso's standard suit, like "cigarettepack," "pullover," and "necklace."
+
+`[assembly:/_pro/characters/templates/sapiensa/char_sapienza_unique.template?/outfit_silviocarusso_actor_v0.entitytemplate].pc_entitytemplate`
+
+# Extracting the Files
+Now that we have found our outfit's TEMP file, we can start extracting the files we need. 
+
+You are advised to create a new folder somewhere that will serve as an output folder for the files we will be extracting. This is only a temporary storage space for these files, as we will be moving them to a Simple Mod Framework folder later on.
+
+With the correct chunk, TEMP and variant selected, right click the filename in the left panel and select `Extract to QN (QuickEntity) JSON`. Select the output folder created in the previous step. This will create a JSON file. You can name it whatever you'd like, but make sure it ends with `.entity.json`.
+
+Since Caruo's outfit is in `chunk1` but our mod will be in `chunk0`, we will also need to extract the depends (which are the assets the outfit uses) and `patch` them to chunk0. Remember that while chunk1 can access chunk0, chunk0 can not access chunk1. If we try to run our mod without including the depends it needs, the game will simply crash.
+
+While you still have the correct file selected in the left box, left-click `Extract Hash Depends` in the right box, then select `Extract Recursive Hash Depends` and extract them to the same folder as the JSON. 

```

Comment:
It may be beneficial to mention that SMF can extract dependencies for you. I'm not sure if it still skips chunk1 though. You'll have to speak to Atampy.